src/test/kotlin/org/jetbrains/research/transformations/anonimization/AnonymizationTest.kt

```diff
@@ -0,0 +1,44 @@
+package org.jetbrains.research.transformations.anonimization
+
+import PythonTransformationsTest
+import com.github.gumtreediff.gen.python.PythonTreeGenerator
+import com.github.gumtreediff.tree.TreeContext
+import org.jetbrains.research.transformations.util.Util.getContentFromFile
+import org.junit.jupiter.api.Assertions.*
+import org.junit.jupiter.params.ParameterizedTest
+import org.junit.jupiter.params.provider.Arguments
+import org.junit.jupiter.params.provider.MethodSource
+import java.io.File
+import java.util.stream.Stream
+
+
+internal class AnonymizationTest : PythonTransformationsTest() {
+
+    companion object {
+        @JvmStatic
+        fun getResourceFolder(): Stream<Arguments> {
+            return Stream.of(
+                Arguments.of(
+                    getResourcePath("in_1.py", ::AnonymizationTest),
+                    getResourcePath("out_1.py", ::AnonymizationTest)
```

Comment:
Shouldn't we automatically get all pairs that satisfy the condition of in-out files using, for example, regular expressions?

Reply:
How? You can not get the resource folder. Or do you mean getting the out file names according to in file names?

Reply:
Hmm I don't know how to do this properly, let's also keep it like this